### ISSUE TITLE
Supply a medial mask

### ISSUE THREAD

**Issue by dfsp-spirit** [dfsp-spirit/brainload#2]
Ship a file containing a medial mask for fsaverage and a function that allows users to apply this mask to data in a single function call, like:

```python
import brainload as bl
morph_data =  ...
masked_moph_data = bl.apply_mask(morph_data, bl.get_fsaverage_medial_mask())
```

**Comment by dfsp-spirit** [comment]
Now that we have support for reading FreeSurfer label files since fabc7d589f7fe537200bf67509c0dd585ffb54d9, maybe there is no need anymore to supply a mask file for fsaverage: such a mask can be generated easily based on the `?h.cortex.label` files, I guess. I will look into this and add a convenience function which does exactly that if the cortex label file is what is typically used to create the mask.

**Comment by dfsp-spirit** [comment]
The mentioned function is available now, see 2611f9411955c7b775b1c22f74b975c7e0d76771. It is called `label_to_mask` and will be part of the next release. Closing this.
